### DISCUSSION TITLE
github教育申请已经三天了为什么不给通过

### DISCUSSION THREAD

**Discussion by 87use** [community/community#145943] [GitHub Education]
### Select Topic Area

Question

### Body

github教育申请已经三天了为什么不给通过

**Comment by daqhris** [comment]
Don't you realize that this is a question that can not be answered by the Github Community?
No one outside of yourself is aware of your communications with Github Education. 
Kindly send them an e-mail or wait patiently for an acceptance or rejection of your application.
